examples/ServoStop/ServoStop.ino

```diff
@@ -0,0 +1,50 @@
+/*
+  Setting servo position to 0 whenever any object distance
+  with the ultrasonic sensor is less than 17cm. Setting Servo position to 90 otherwise
+  by Madhur Dixit https://github.com/Chester-King
+
+  You can see the detailed description and required circuit for the sketch at https://gist.github.com/Chester-King/fc19b83777ff629b6708731f09eeaa6c  
```

Comment:
Add these files to the example folder. The link would be broken if you deleted the gist or your GitHub account.

Reply:
Files added to examples folder and links modified accordingly.